### PR TITLE
Update tile load test script and add scripts for hompage load, facility download, and facility POST

### PR DESCRIPTION
**Pull request by jwalgran**
## Overview

Update tile load test script and add scripts for hompage load, facility download, and facility POST

Connects #1776

## Testing Instructions

**NOTE** these scripts have been successfully used on staging so this can be a cursory review.

* Register an account on staging and generate an API token
* Obtain the `CLIENT_KEY` used on staging
* `cd` into the `load-tests` directory.
* Verify that you can run the new load test commands added to the README in the PR

## Checklist

- [x] `fixup!` commits have been squashed
- [x] CI passes after rebase
- [x] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines

### CODE REVIEW

File: load-tests/browse_homepage.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    // Wait up to 5 seconds so that the iterations aren't exactly overlapping
+    sleep(Math.random() * 5);
+
+    group('https://staging.openapparel.org/', function () {
+        responses = http.batch([
+            buildGet(`${rootUrl}/web/environment.js`),
+            buildGet(`${rootUrl}/static/css/2.037bc208.chunk.css`),
+            buildGet(`${rootUrl}/static/css/main.cfc7f5ef.chunk.css`),
+            buildGet(`${rootUrl}/static/js/2.b90b8892.chunk.js`),
+            buildGet(`${rootUrl}/static/js/main.058b1d83.chunk.js`),
```
Comment (jwalgran) [inline load-tests/browse_homepage.js:61]: These are going to be different for each release. I think the way to get these dynamically is to search the HTML returned from a request to `/`

**Review by emilyhu0106 — COMMENTED** [review]
The tests looks very good! Wondering if below is the expected output for the facility POSTing test?
```
  execution: local
     script: /scripts/post_facility.js
     output: -

  scenarios: (100.00%) 1 scenario, 5 max VUs, 10m30s max duration (incl. graceful stop):
           * default: 5000 iterations shared among 5 VUs (maxDuration: 10m0s, gracefulStop: 30s)


running (03m34.3s), 0/5 VUs, 5000 complete and 0 interrupted iterations
default ✓ [======================================] 5 VUs  03m34.6s/10m0s  5000/5000 shared iters

     ✗ is status 2xx
      ↳  0% — ✓ 0 / ✗ 5000

     checks.....................: 0.00%   ✓ 0         ✗ 5000
     data_received..............: 2.0 MB  9.2 kB/s
     data_sent..................: 1.3 MB  6.2 kB/s
     http_req_blocked...........: avg=388.49µs min=25.3µs   med=35.4µs   max=342.4ms  p(90)=76.71µs  p(95)=113.3µs 
     http_req_connecting........: avg=88.7µs   min=0s       med=0s       max=92.85ms  p(90)=0s       p(95)=0s      
     http_req_duration..........: avg=213.29ms min=144.79ms med=199.96ms max=4.64s    p(90)=232.73ms p(95)=254.86ms
     http_req_failed............: 100.00% ✓ 5000      ✗ 0   
     http_req_receiving.........: avg=301.9µs  min=69.3µs   med=192.5µs  max=22.68ms  p(90)=439.34µs p(95)=664.28µs
     http_req_sending...........: avg=326.84µs min=118.7µs  med=287.7µs  max=1.8ms    p(90)=496.43µs p(95)=603.02µs
     http_req_tls_handshaking...: avg=134.93µs min=0s       med=0s       max=138.33ms p(90)=0s       p(95)=0s      
     http_req_waiting...........: avg=212.66ms min=143.8ms  med=199.35ms max=4.64s    p(90)=232.06ms p(95)=254.3ms 
     http_reqs..................: 5000    23.327565/s
     iteration_duration.........: avg=214.48ms min=179.94ms med=200.66ms max=4.98s    p(90)=233.45ms p(95)=255.38ms
     iterations.................: 5000    23.327565/s
     vus........................: 5       min=5       max=5 
     vus_max....................: 5       min=5       max=5 
```

I also did some readings about k6 and load tests, reviewing this PR is helpful!

**Comment by jwalgran** [comment]
Sorry for the delayed response on your comment. The `POST` test output your shared shows that all the requests responded with a status other than 200, which is not correct.

That script does require that `TOKEN` is set to a valid API token from staging. Did you run the command with a token from a staging account? If not, I can share my staging token with you that is configured for unlimited staging API access

```
 TOKEN=abc123-some-real-token-here \
docker-compose -f docker-compose.yml \
  run --rm k6 run -u 5 -i 5000  /scripts/post_facility.js
```

**Comment by emilyhu0106** [comment]
I just ran this test twice with a staging API Token. 

The first time failed in the middle of the tests, probably due to api limits. The result is as below:
```
running (10m02.3s), 0/5 VUs, 1463 complete and 0 interrupted iterations
default ✗ [==========>---------------------------] 5 VUs  10m03.0s/10m0s  1463/5000 shared iters

     ✗ is status 2xx
      ↳  96% — ✓ 1405 / ✗ 58
```

Then I gave myself unlimited api calls, ran it again. It was complete but failed
```
running (03m09.4s), 0/5 VUs, 5000 complete and 0 interrupted iterations
default ✓ [======================================] 5 VUs  03m09.6s/10m0s  5000/5000 shared iters

     ✗ is status 2xx
      ↳  0% — ✓ 0 / ✗ 5000
```

Should I try with another API token?

**Comment by jwalgran** [comment]
I clicked the "close" button by accident.

**Comment by jwalgran** [comment]
I scheduled a pairing session on Friday May 6 to look at this together.

**Review by emilyhu0106 — APPROVED** [review]
All tests are passed after the pairing. It was related to the API blocks that can be updated in the Admin Dashboard. Thank you Justin!

**Comment by jwalgran** [comment]
Thanks for the review.

**Pull request merged**